### PR TITLE
✨ Add SplashPage to components

### PR DESCRIPTION
**Pull request by saschahaselhuhn**

### CODE REVIEW

File: components/splashpage/Splashpage.module.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.gridContainer {
+  height: 100vh;
+  display: grid;
+  justify-content: center;
+}
+
+img {
```
Comment (lmachens) [inline components/splashpage/Splashpage.module.css]: Make sure to not affect other images
```suggestion
.gridContainer img
```

**Review by mrana17 — APPROVED** [review]
super 👍

**Review by lmachens — COMMENTED** [review]
Good job.
I would extract texts to social media from the links to support text browser and crawler.

**Review by RonjaMeinecke — COMMENTED** [review]
GJ 🚀

**Pull request merged**